pkg/rpcclient/wsclient_test.go

```diff
@@ -52,6 +52,7 @@ func TestWSClientSubscription(t *testing.T) {
 	aerCh := make(chan *state.AppExecResult)
 	ntfCh := make(chan *state.ContainedNotificationEvent)
 	ntrCh := make(chan *result.NotaryRequestEvent)
+	mempoolCh := make(chan *result.MempoolEvent)
```

Comment:
I can't add comment to a proper place, but TestWSClientEvents should also be extended.

Reply:
As long as TestWSFilteredSubscriptions.